var/spack/repos/builtin/packages/armcimpi/package.py

```diff
@@ -26,6 +26,8 @@ class ArmciMpi(AutotoolsPackage):
     variant("shared", default=True, description="Builds a shared version of the library")
     variant("progress", default=False, description="Enable asynchronous progress")
 
+    provides("armci")
```

Comment:
You shouldn't need a virtual package for `armci` unless you're adding a second provider for it.

Reply:
If Global Arrays (GA) becomes a separate Spack package, then it will be the second provided.  GA is currently not separate from NWChem but it can be and there are some apps that use it directly.  I intend to address this eventually but have higher priorities for now.

Reply:
Ok, fyi Global Arrays has a package, https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/globalarrays/package.py

Reply:
Ah sorry. I guess I searched for "ga" and missed it.

Reply:
a subsequent PR will reconcile the globalarrays and nwchem packages.  see https://github.com/spack/spack/commit/11a943c66846ed8c32103a17b22b2bcae8950d28 for details.